### PR TITLE
Add some fixed seed nodes.

### PR DESCRIPTION
**Pull request by domob1812**
They are based on seed.namecoin.domob.eu as well as a few .onion addresses explicitly contributed/known.

See also https://forum.namecoin.info/viewtopic.php?f=8&t=2434.

### CODE REVIEW

**Comment by phelixbtc** [comment]
Untested ACK. Will try and test but it might take another two weeks, sorry.

File: contrib/seeds/generate-seeds.py [hunk 1]
```diff
@@ -127,10 +127,10 @@ def main():
     g.write(' * IPv4 as well as onion addresses are wrapped inside a IPv6 address accordingly.\n')
     g.write(' */\n')
     with open(os.path.join(indir,'nodes_main.txt'),'r') as f:
-        process_nodes(g, f, 'pnSeed6_main', 8333)
+        process_nodes(g, f, 'pnSeed6_main', 8334)
     g.write('\n')
     with open(os.path.join(indir,'nodes_test.txt'),'r') as f:
-        process_nodes(g, f, 'pnSeed6_test', 18333)
+        process_nodes(g, f, 'pnSeed6_test', 18334)
```
Comment (JeremyRand) [inline contrib/seeds/generate-seeds.py]: Just checking to make sure I understand this change; it looks like you're changing the Bitcoin peer port to the Namecoin peer port.  Am I correct, or is this something else?

Comment (domob1812) [inline contrib/seeds/generate-seeds.py]: Yes, exactly.  Just changing the (default) port.

**Comment by phelixbtc** [comment]
Compiles on windows and seems to work: `2015-11-19 11:15:43 90 addresses found from DNS seeds`

**Comment by domob1812** [comment]
Any news on this, can I merge it?

**Comment by phelixbtc** [comment]
If it was not clear from above: ACK from me :)

**Comment by domob1812** [comment]
Not sure why this is messed up in the view with respect to the commits (after rebasing onto current master), but I've merged this now.  Will backport to 0.12 as well.

**Pull request merged**